orion/pkg/controllers/session_controller.go

```diff
@@ -0,0 +1,85 @@
+package controllers
+
+import (
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/domains"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/services"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+func GetAllSessionsByClassId(c *gin.Context) {
+	classId := c.Param("classId")
+
+	sessionList, err := services.SessionService.GetAllByClassId(classId)
+	if err != nil {
+		c.String(http.StatusInternalServerError, err.Error())
+	} else {
+		c.JSON(http.StatusOK, sessionList)
+	}
+	return
+}
+
+func GetSessionById(c *gin.Context) {
+	// Incoming parameters
+	id := ParseParamId(c)
+
+	session, err := services.SessionService.GetBySessionId(id)
+	if err != nil {
+		c.String(http.StatusNotFound, err.Error())
+	} else {
+		c.JSON(http.StatusOK, session)
+	}
+	return
+}
+
+func CreateSession(c *gin.Context) {
+	// Incoming JSON
+	var sessionJson domains.Session
+	c.BindJSON(&sessionJson)
+
+	if err := sessionJson.Validate(); err != nil {
+		c.String(http.StatusBadRequest, err.Error())
+		return
+	}
+
+	err := services.SessionService.Create(sessionJson)
+	if err != nil {
+		c.String(http.StatusInternalServerError, err.Error())
+	} else {
+		c.JSON(http.StatusOK, nil)
```

Comment:
Also from last week's PR, if you are not returning a body, just use c.Status(OK)

Reply:
*bump* please use c.Status